lib/resource-mapper.js

```diff
@@ -37,49 +44,87 @@ class ResourceMapper {
     }
   }
 
-  get rootPath () {
-    return this._rootPath
+  // Returns the URL of the given HTTP request
+  getRequestUrl (req) {
```

Comment:
Allright, but then I don't quite understand the purpose of this PR, as wasn't the intention that it would bring it back to the original interface?

Reply:
Yes, but this one has been ruled harmless too.

The idea is that this class will be used for all tasks URL and file, and "obtain the full URL" was used in a couple of places, so we better want that done correctly.